amqp_connection.c

```diff
@@ -1238,6 +1248,39 @@ static PHP_METHOD(amqp_connection_class, setVerify)
 }
 /* }}} */
 
+/* {{{ proto amqp::getSaslMethod()
+get sasl method */
+static PHP_METHOD(amqp_connection_class, getSaslMethod)
+{
+	PHP5to7_READ_PROP_RV_PARAM_DECL;
+	PHP_AMQP_NOPARAMS();
+	PHP_AMQP_RETURN_THIS_PROP("sasl_method");
+}
+/* }}} */
+
+/* {{{ proto amqp::setSaslMethod(mixed method)
+set sasl method */
+static PHP_METHOD(amqp_connection_class, setSaslMethod)
+{
+	long method;
+
+	/* Get the port from the method params */
+	if (zend_parse_parameters(ZEND_NUM_ARGS() TSRMLS_CC, "l", &method) == FAILURE) {
+		return;
+	}
+
+	/* Check the method value */
+	if (method != AMQP_SASL_METHOD_PLAIN && method != AMQP_SASL_METHOD_EXTERNAL) {
+		zend_throw_exception(amqp_connection_exception_class_entry, "Invalid sasl method given. Method must be AMQP_SASL_METHOD_PLAIN or AMQP_SASL_METHOD_EXTERNAL.", 0 TSRMLS_CC);
```

Comment:
Message should be `Invalid SASL method given …`

Reply:
@lstrojny 
addressed